internal/entities.go

```diff
@@ -12,6 +12,18 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+type (
+	ParsedEntityListData struct {
+		Title string `json:"title"`
+	}
+
+	NamedItemWithType struct {
+		Title string `json:"title"`
```

Comment:
Pehaps there are more properties that would be interesting in these structs. `title` is of interest but for a dimension I guess which fields that are used would also be good to output, and e.g. expression for a measure. This would require separate structs and more refactoring though.